ocaml/xapi/xapi_pif.ml

```diff
@@ -362,7 +363,7 @@ let find_or_create_network (bridge : string) (device : string) ~__context =
         Db.Network.create ~__context ~ref:net_ref ~uuid:net_uuid
           ~current_operations:[] ~allowed_operations:[]
           ~name_label:(Helpers.choose_network_name_for_pif device)
-          ~name_description:"" ~mTU:1500L ~purpose:[] ~bridge ~managed:true
+          ~name_description:"" ~mTU:1500L ~purpose:[] ~bridge ~managed
```

Comment:
Is the `managed` field in `network` object a duplication of the field `managed` of `PIF` objects?
In other words, if a PIF is managed, its connected network must be managed?

Reply:
I believe they are separate, one is for the network bridge which connects things together, while the other one is for configuring the interface (bring the interface up/down, etc)